### DISCUSSION TITLE
My awakening POE is broken

### DISCUSSION THREAD

**Discussion by moonkehh** [SnosMe/awakened-poe-trade#1434] [Ideas]
When I click on trade it doesnt load on a browser? It worked once and then never again. What is the issue?

**Comment by moonkehh** [comment]
Also, I  should say i checked out the https://snosme.github.io/awakened-poe-trade/nothing-happens.html
none of these problems are my issue. What happened was it opened in chrome and worked then I used opera and it loaded there for some stupid reason when i dont even use it so i uninstallled opera and now it doesnt load anywhere.  Is this a config issue?

**Reply by SnosMe** [reply]
Make the default browser Edge and then make Chrome again, this is os issue, not the app

**Comment by moonkehh** [comment]
Thank you it worked!
